From d1fffcc3366211b7aacf02de2a78936092178e5c Mon Sep 17 00:00:00 2001
From: Eric Naim <dnaim@cachyos.org>
Date: Sun, 1 Jun 2025 21:11:36 +0700
Subject: [PATCH 48/48] Revert "drm/amdgpu: Don't use doorbells for SDMA 5.2"

This reverts commit c12920a38a699e8cf9f9c383e73259eb5ef1b9a0.

This seems to break Ollama, etc.

Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 drivers/gpu/drm/amd/amdgpu/sdma_v5_2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/sdma_v5_2.c b/drivers/gpu/drm/amd/amdgpu/sdma_v5_2.c
index cbe734ffa..2b39a03ff 100644
--- a/drivers/gpu/drm/amd/amdgpu/sdma_v5_2.c
+++ b/drivers/gpu/drm/amd/amdgpu/sdma_v5_2.c
@@ -1339,7 +1339,7 @@ static int sdma_v5_2_sw_init(struct amdgpu_ip_block *ip_block)
 	for (i = 0; i < adev->sdma.num_instances; i++) {
 		ring = &adev->sdma.instance[i].ring;
 		ring->ring_obj = NULL;
-		ring->use_doorbell = false;
+		ring->use_doorbell = true;
 		ring->me = i;
 
 		DRM_INFO("use_doorbell being set to: [%s]\n",
-- 
2.49.0

